esp/esp/program/modules/handlers/onsiteclasslist.py

```diff
@@ -149,8 +149,9 @@ def counts_status(self, request, tl, one, two, module, extra, prog):
     @needs_onsite
     def rooms_status(self, request, tl, one, two, module, extra, prog):
         resp = HttpResponse(mimetype='application/json')
-        data = ClassSection.objects.filter(status__gt=0, parent_class__status__gt=0, parent_class__parent_program=prog).select_related('resourceassignment__resource__name').values_list('id', 'resourceassignment__resource__name', 'resourceassignment__resource__num_students')
-        simplejson.dump(list(data), resp)
+        data = ClassSection.objects.filter(status__gt=0, parent_class__status__gt=0, parent_class__parent_program=prog).select_related('resourceassignment__resource__name').values_list('id', 'resourceassignment__resource__name', 'resourceassignment__resource__num_students', 'resourceassignment__resource__res_type__name')
+        data = [d for d in data if d[3] == 'Classroom']
```

Comment:
It's not noticeably slower on my devserver (for the Splash 2014 grid) in the current state compared to not making any modifications, and I also couldn't think of a way to do the restriction in the DB (since we need to restrict the returned tuples in the list, not the section objects) without using ResourceAssignment.objects.[...] which is painfully slow for some reason (unless I just messed up and what I tried outright doesn't work). If you have an alternate solution, I can try it out, but I don't think it'll make a big difference anyway.